src/commands/logStream/RevisionListStep.ts

```diff
@@ -12,19 +12,23 @@ import { localize } from "../../utils/localize";
 import { IStreamLogsContext } from "./IStreamLogsContext";
 
 export class RevisionListStep extends AzureWizardPromptStep<IStreamLogsContext> {
+    private revisions: Revision[] | undefined;
+
     public async prompt(context: IStreamLogsContext): Promise<void> {
         const placeHolder: string = localize('selectRevision', 'Select a revision');
-        context.revision = (await context.ui.showQuickPick(this.getPicks(context), { placeHolder })).data;
+        context.revision = (await context.ui.showQuickPick(await this.getPicks(context), { placeHolder })).data;
```

Comment:
Generally you don't want to await `this.getPicks` in `showQuickPick`. `showQuickPick` is designed to handle promises so that it will show a loading bar while the promise resolves. If you await it, there'll be a small gap where the quickpick UI doesn't show up while it's loading the picks.

Reply:
whoops accidently added that ahaha.